wordpress-mobile/WordPress-Android
# WP Stories integration - Activity launch

This PR builds on top of #11864 

This PR adds the Activity launching code so it's accessible from the FAB.

The basic Activity linking functionality implemented also in this PR in this commit 02e7de8 (tapping on the main screen FAB should show a third option to create a Story and trigger the composing functionality in the Stories library).

#### To test:
1. checkout the branch and run 
```
git submodule init 
git submodule update
```
or for a fresh clone, you can also do `git clone --recurse-submodules git@github.com:wordpress-mobile/WordPress-Android.git`

2. build
3. run
4. once you land on the main screen and provided you have a site, tap on the FAB and then choose the third option (Story).
5. the camera capture mode from the Stories library should open up, and permissions will be requested accordingly.


To test:

PR submission checklist:
